dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/dd42e77b-24ce-485d-8146-ee6c96d5b454.json

```diff
@@ -0,0 +1,29 @@
+{
+  "sourceId": "e094cb9a-26de-4645-8761-65c0c425d1de",
+  "sourceSpecificationId": "dd42e77b-24ce-485d-8146-ee6c96d5b454",
+  "specification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Stripe Source Spec",
+    "type": "object",
+    "required": ["client_secret", "account_id", "start_date"],
+    "additionalProperties": false,
+    "properties": {
+      "client_secret": {
+        "type": "string",
+        "pattern": "^sk_(live|test)_[a-zA-Z0-9]+$",
+        "description": "Stripe API key (starts with 'sk_live_' in production, see https://dashboard.stripe.com/apikeys)."
+      },
+      "account_id": {
+        "type": "string",
+        "pattern": "^acct_[a-zA-Z0-9]+$",
+        "description": "Your Stripe account ID (starts with 'acct_', see https://stripe.com/docs/api/accounts)."
+      },
+      "start_date": {
+        "type": "string",
+        "default": "2009-01-01T00:00:00Z",
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$",
+        "description": "A UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated."
```

Comment:
This actually supports any date time in https://dateutil.readthedocs.io/en/stable/parser.html, which is fairly permissive. I'm a little worried that people will be confused or enter in ambiguous data if we aren't more prescriptive (requiring at least timezone). Does this seem like a reasonable format?